tools/cli/pkg/command/orchestration.go

```diff
@@ -395,7 +395,8 @@ func (cmd *OrchestrationCommand) cancelOrchestration(orchestrationID string) err
 // Currently only orchestrations of type "kyma upgrade" are supported,
 // and the type is not reflected in the StatusResponse object
 func orchestrationType(obj interface{}) string {
-	return "kyma upgrade"
+	sr := obj.(orchestration.StatusResponse)
+	return string(sr.Type)
```

Comment:
I think the representation values are not too user friendly. How about switching here based on `.Type`, and return `"kyma upgrade"` or `"cluster upgrade"`?

And this function can be exposed to the Go template used above, and invoked to also print the nicer looking value instead of the raw data. See the `orchestrationTarget` function as an example.